### PR TITLE
Select Library to import into

### LINKED ISSUE #6177: Select library during import
Would it be possible to select the library into which the entry is imported?

I often have several libraries open at the same time, and I have to switch to the desired library before importing (something that I often forget).

### PR DESCRIPTION
**Pull request by MaryJml**
Fixes #6177 

[#6177 ]Select library during import 
截屏2022-11-24 22 13 31

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/resources/l10n/JabRef_en.properties [hunk 1]
```diff
@@ -2514,3 +2514,5 @@ Keep\ old\ entry=Keep old entry
 No\ entries\ corresponding\ to\ given\ query=No entries corresponding to given query
 
 Use\ the\ field\ FJournal\ to\ store\ the\ full\ journal\ name\ for\ (un)abbreviations\ in\ the\ entry=Use the field FJournal to store the full journal name for (un)abbreviations in the entry
+
+Library\ to\ import\ into=Library to import into
```
Comment (ThiloteE) [inline src/main/resources/l10n/JabRef_en.properties:2518]: How about: "`Import into:`" or "`Import into following library:`"?

**Comment by calixtus** [comment]
@MaryJml hey, thanks for the follow-up. Your additions look mostly good to me already. Sadly we have no clean solution for the database changed asterisk until we rework the undomanager, so we still have to live with the call to JabRefGui, so please undo the comment.

Also before merging we probably have to implement the workaround mentioned in ImportEntriesView is suggested.

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java [hunk 2]
```diff
@@ -102,6 +105,29 @@ private void initialize() {
...
+                    if (dbOpt.isEmpty()) {
+                        return "Untitled";
+                    }
+
+                    return dbOpt.get();
+                })
+                .install(libraryListView);
+        viewModel.selectedDbProperty().bind(libraryListView.getSelectionModel().selectedItemProperty());
+
+        // FixMe: getActiveDatabase result object is different from the ones in getOpenDatabases.
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java:126]: @calixtus  Since we merge PR #9450  this comment is no longer valid

**Comment by Siedlerchr** [comment]
For me the asterisk is shown correctly after importing into the library at the library tab.

**Comment by Siedlerchr** [comment]
We definitely need to test this with shared databases

**Comment by Siedlerchr** [comment]
Tested with a shared database, works fine!
grafik

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**